nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -77,6 +77,8 @@
 
 - Cinnamon has been updated to 6.4.
 
+- `services.avahi.ipv6` now defaults to true.
```

Comment:
How does it interact with `networking.enableIPv6`? Maybe the option should default to `networking.enableIPv6`?